.github/workflows/advanced.yml

```diff
@@ -701,6 +702,11 @@ jobs:
           ref: ${{ env.FFMPEG_BRANCH }}
           path: ${{ env.SOURCE_PATH }}/${{ env.FFMPEG_VERSION }}
 
+      - name: Patch FFmpeg
+        shell: bash
+        run: |
+          .github/scripts/ffmpeg/patch.sh
```

Comment:
Should FFmepg fixed version be used? Originally I had also a patch however with new version of FFmpeg it was not compatible.